package.json

```diff
@@ -189,7 +189,6 @@
     "@eslint/eslintrc@^2.0.1": "patch:@eslint/eslintrc@npm%3A2.0.2#./.yarn/patches/@eslint-eslintrc-npm-2.0.2-d308674d86.patch",
     "@fortawesome/fontawesome-free@^5.13.0": "patch:@fortawesome/fontawesome-free@npm%3A5.13.0#./.yarn/patches/@fortawesome-fontawesome-free-npm-5.13.0-f20fc0388d.patch",
     "@keystonehq/bc-ur-registry@^0.5.0-alpha.5": "patch:@keystonehq/bc-ur-registry@npm%3A0.5.0-alpha.5#./.yarn/patches/@keystonehq-bc-ur-registry-npm-0.5.0-alpha.5-b95c7992a6.patch",
-    "@lavamoat/lavapack@^3.1.0": "patch:@lavamoat/lavapack@npm%3A3.1.0#./.yarn/patches/@lavamoat-lavapack-npm-3.1.0-34c65d233b.patch",
```

Comment:
Interesting, this patch was needed to remove absolute file paths from the bundle, but it looks like that was fixed upstream (relative file paths are used instead). Nice, one less patch needed!

Could you remove the patch file as well? It looks like it's still present in the `.yarn/patches` directory

Reply:
I have just double checked to confirm that the absolute paths are gone. They're all relative paths now, so this patch is definitely safe to remove.